store/sql_user_store.go

```diff
@@ -1195,6 +1195,7 @@ func (us SqlUserStore) performSearch(searchQuery string, term string, searchType
 		searchQuery = strings.Replace(searchQuery, "SEARCH_CLAUSE", "", 1)
 	} else if utils.Cfg.SqlSettings.DriverName == model.DATABASE_DRIVER_POSTGRES {
 		term = term + ":*"
+		searchType = convertMySQLFullTextColumnsToPostgres(searchType)
```

Comment:
Can a SQL injection attack happen here?  not sure if serachType is input from the user or not.